Makefile

```diff
@@ -1,27 +0,0 @@
-# (c) Copyright 2019-2023 OLX
-DOCKER_REGISTRY ?= ghcr.io
-DOCKER_ORG ?= olxgroup-oss
-DALI_IMAGE_NAME ?= $(DOCKER_REGISTRY)/$(DOCKER_ORG)/dali/dali
-BUILD_NUMBER ?= $(shell ./scripts/get-current-version.sh)
-
-ifeq ($(REVISION),)
-	VERSION_TAG ?= $(BUILD_NUMBER)
-else
-	VERSION_TAG ?= $(REVISION)-preview
-endif
-
-SUDO := $(shell docker info >/dev/null 2>&1 || echo sudo)
-
-.PHONY: test run docker-build docker-publish
-
-test:
```

Comment:
Why entirely removing the Makefile? The `test`, `run` and `docker-build` are still useful for the local development.

Reply:
Agreed, and the docker-compose file should also be restored from when it was removed? There is reference to it in the README

Reply:
Let's restore it. It seems that it's handling the integration tests for now.